core/src/main/java/com/sequenceiq/cloudbreak/job/stackpatcher/ExistingStackPatcherJob.java

```diff
@@ -70,23 +70,26 @@ protected void executeTracedJob(JobExecutionContext context) throws JobExecution
             StackPatchType stackPatchType = stackPatchTypeConverter.convertToEntityAttribute(stackPatchTypeName);
             if (stackPatchType == null || StackPatchType.UNKNOWN.equals(stackPatchType)) {
                 String message = String.format("Stack patch type %s is unknown", stackPatchTypeName);
-                unscheduleAndFail(message, context, stack, stackPatchType);
+                unscheduleAndFailJob(message, context, stack, stackPatchType);
             } else {
                 Optional<ExistingStackPatchService> optionalExistingStackPatchService = getStackPatchServiceForType(stackPatchType);
                 if (optionalExistingStackPatchService.isEmpty()) {
                     String message = "No stack patcher implementation found for type " + stackPatchType;
-                    unscheduleAndFail(message, context, stack, stackPatchType);
+                    unscheduleAndFailJob(message, context, stack, stackPatchType);
                 } else {
-                    applyStackPatch(optionalExistingStackPatchService.get(), stack);
+                    boolean success = applyStackPatch(optionalExistingStackPatchService.get(), stack);
+                    if (success) {
+                        unscheduleJob(context);
```

Comment:
could you add log here and/or in `unscheduleJob`?

Reply:
Added log message to unscheduleJob